Week4/RecyclerView/app/src/main/res/layout/cell_tvshow.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
You don't need this ConstraintLayout, just make the CardView the root of your hierarchy.